tekton/resources/nightly-tests/eventlistener.yaml

```diff
@@ -65,6 +65,18 @@ spec:
     - ref: trigger-to-deploy-test-tekton-project
     template:
       ref: tekton-dashboard-nightly-test-s390x
+  - name: catalog-nightly-test-trigger-s390x
+    interceptors:
+    - cel:
+        filter: >-
+          'trigger-template' in body &&
+           body['trigger-template'] == 'catalog' &&
+           'arch' in body.params.target &&
+           body.params.target.arch == 's390x'
```

Comment:
Once https://github.com/tektoncd/plumbing/pull/956 is merged, I would like to move these to use `Triggers` and `TriggerGroups` as well - we can have one group for each arch